app/views/learn_more/show.html.erb

```diff
@@ -34,20 +34,7 @@
       <div class="w3-panel">
         <%= link_to "Back to Welcome", "/", class: "w3-btn w3-light-green w3-block" %>
       </div>
-      <div class="w3-panel">
-        <script src="https://apis.google.com/js/platform.js" async defer></script>
-        <%= link_to "Login with Google", "/auth/google_oauth2", class: "w3-btn w3-blue-grey w3-block" %>
-      </div>
-      <div class="w3-panel w3-pale-blue w3-block w3-center">
-        <h4 >Login with Google to:</h4>
-        <p class="w3-justify w3-center ">
-          - Set up a garden - <br>
-          - Track your sensor data - <br>
-          - Connect with your community - <br>
-          - Track your garden's carbon impact - <br>
-          - Track the health of your plants and soil -
-        </p>
-      </div>
+      <%= render 'shared/login_with_google_button' %>
```

Comment:
Did this change the styling at all?

Reply:
I checked it on localhost and it looked good. Admittedly I have not been exposed to the FE as much as you and Luke! So you may notice something I didn't.

Reply:
It all looks the same!  You're a wizard Angela :)